tests/testthat/test-session.R

```diff
@@ -208,3 +208,24 @@ test_that("logs", {
   expect_match(log$message[1], "Hello again")
   expect_equal(names(log), c("timestamp", "level", "message"))
 })
+
+
+test_that("can pass additional parameters", {
+  user_agent_string <- "my_user_agent"
+  ap <- list(phantomjs.page.settings.userAgent = unbox("my_user_agent"))
+  s <- session$new(port = phantom$port
+                   , additional_parameters = ap
```

Comment:
Can you please reformat this like this:
```r
s <- session$new(port = phantom$port,  additional_parameters = ap)
```
or if it longer than 75 characters, like this:
```r
s <- session$new(
  port = phantom$port,
  additional_parameters = ap
)
```

and also the ones below? I know that leading comma is common is JS, but not so much in R, and I don't do it in this package. Thanks!